s3tests/functional/test_s3.py

```diff
@@ -5351,9 +5351,15 @@ def test_multipart_copy_invalid_range():
     bucket, key = _create_key_with_random_content('source', size=5)
     upload = bucket.initiate_multipart_upload('dest')
     e = assert_raises(boto.exception.S3ResponseError, copy_part, bucket.name, key.name, bucket, 'dest', upload.id, 0, 0, 21)
-    eq(e.status, 400)
-    eq(e.reason, 'Bad Request')
-    eq(e.error_code, 'InvalidArgument')
+    # on RHEL we get response 416, check bugzilla 1501005
+    valid_status = [400, 416]
+    if not any(s == e.status for s in valid_status):
```

Comment:
`if e.status not in valid_status`:
will also work here

Reply:
better!, will update